src/Renci.SshNet/Renci.SshNet.csproj

```diff
@@ -7,44 +7,18 @@
     <AssemblyOriginatorKeyFile>../Renci.SshNet.snk</AssemblyOriginatorKeyFile>
     <LangVersion>6</LangVersion>
     <SignAssembly>true</SignAssembly>
-    <TargetFrameworks>net35;net40;net472;netstandard1.3;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net472;netstandard2.0;net7.0</TargetFrameworks>
   </PropertyGroup>
 
-  <!--
-  <PropertyGroup Condition=" '$(VisualStudioVersion)' == '16.0' ">
-    <TargetFrameworks>net35;net40;netstandard1.3;netstandard2.0;netstandard2.1</TargetFrameworks>
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net472' ">
```

Comment:
We can probably remove lots of the symbols as the corresponding API's are now available for all supported target frameworks. This can be done in a follow-up PR of course.

Reply:
Yes I've seen some places but I think it should be next PR only for this.

Reply:
I agree